Lib/multiprocessing/forkserver.py

```diff
@@ -56,7 +56,7 @@ def connect_to_new_process(self, fds):
         '''Request forkserver to create a child process.
 
         Returns a pair of fds (status_r, data_w).  The calling process can read
-        the child process's pid and (eventually) its returncode from status_r.
+        the child process\'s pid and (eventually) its returncode from status_r.
```

Comment:
I don't know if you changed this because it tripped your text editor, but this escaping is unnecessary inside a triple-quoted string.

Reply:
Your guess is correct - it made xemacs unable to tell what was quoted after it, which got especially inconvenient when trying to make sure everything was indented correctly.